gitlab_runner_config.py

```diff
@@ -53,20 +56,39 @@ def generate_tags(instance, executor_type="", env=None):
     These tags are specified by runner configs and used by CI specs to run jobs
     on the appropriate host.
     """
-
-    tags = identifying_tags(instance)
-    if executor_type:
-        tags.append(executor_type)
-    if env:
-        tags += [os.environ[e] for e in env if e in os.environ]
+    properties = {
+        "hostname": HOSTNAME,
+        "executor_type": executor_type,
+        "instance": instance,
+        "env": []
+        }
+    if env: 
+        properties["env"] += [os.environ[e] for e in env if e in os.environ]
+    # if executor is batch, gather some more system info for tags
     if executor_type == "batch":
```

Comment:
We can actually just remove this whole block for adding the scheduler--we'll make that the responsibility of `capture_tags`